src/ralph/utils.py

```diff
@@ -84,3 +86,14 @@ def now():
     """Return the current UTC time in ISO format"""
 
     return datetime.datetime.now(tz=datetime.timezone.utc).isoformat()
+
+
+def get_dict_value_from_path(dict_: dict, path: list[str]):
+    """Get a nested dictionary value using an array of keys representing the path to the value."""
```

Comment:
```suggestion
    """Gets a nested dictionary value using an array of keys representing the path to the value."""
```

Reply:
Thanks) This will make exactly 100 character for this line)